core/src/main/java/io/questdb/griffin/SqlUtil.java

```diff
@@ -66,6 +66,7 @@
 import static io.questdb.std.datetime.millitime.DateFormatUtils.PG_DATE_MILLI_TIME_Z_FORMAT;
 import static io.questdb.std.datetime.millitime.DateFormatUtils.PG_DATE_Z_FORMAT;
 
+@SuppressWarnings("unused")
```

Comment:
lets suppress method by method instead, which would include comments as to why method is suppressed